common/djangoapps/course_modes/models.py

```diff
@@ -835,6 +836,22 @@ def is_eligible_for_certificate(cls, mode_slug, status=None):
 
         return mode_slug not in ineligible_modes
 
+    @classmethod
+    def get_certificate_relevant_modes(cls):
+        """
+        Wrapper for CERTIFICATE_RELEVANT_MODES.
+
+        When EXTEND_CERTIFICATE_RELEVANT_MODES_WITH_HONOR_FLAG is enabled we want to add HONOR mode.
+        We may want that when using the HONOR courses in the programs and need to notify credentials
+        about such course certificates.
+        """
+        cert_relevant_modes = cls.CERTIFICATE_RELEVANT_MODES
+
+        if EXTEND_CERTIFICATE_RELEVANT_MODES_WITH_HONOR_FLAG.is_enabled():
+            cert_relevant_modes += [cls.HONOR]
+
+        return cert_relevant_modes
+
```

Comment:
This seems redundant with the `is_eligible_for_certificate`, which already has a setting (`DISABLE_HONOR_CERTIFICATES`) to control whether this Open edX instance should care about `Honor` mode certificates. 

Was there a reason the existing functionality wasn't sufficient in the `notify_credentials` work flow?